hack/run-ci-e2e-test.sh

```diff
@@ -110,6 +110,16 @@ if  ! oc get deploy/windows-machine-config-operator -n $WMCO_DEPLOY_NAMESPACE >
       retries+=1
   done
   wmco_deployed_by_script=true
+
+  # OCPBUGS-23121: For AWS, test the operator installed in a different namespace after a successful deployment in the
+  # provided namespace
+  platform="$(oc get infrastructure cluster -ojsonpath={.spec.platformSpec.type})"
+  if [[ "$platform" = "AWS" ]]; then
+    cleanup_WMCO $OSDK
+    export WMCO_DEPLOY_NAMESPACE="${WMCO_DEPLOY_NAMESPACE}-customized"
+    run_WMCO $OSDK
+    get_WMCO_logs
+  fi
```

Comment:
Do we really need to add this test?
Our test suite is so complicated already and this is a true edge case

Reply:
I had mixed feelings about the value of adding e2e coverage for this corner case. So, I proposed the test for one platform only (AWS).  

Happy to remove it from the PR if that is the consensus. @openshift/openshift-team-windows-containers please chime in.

Reply:
I am fine with removing it.

Reply:
Removed the test commits.